tests/testthat/test-petpeese.R

```diff
@@ -0,0 +1,149 @@
+context("Meta Analysis - PET-PEESE")
```

Comment:
Is it possible to add the (commented out) R code and output of these results in the unit tests? Not required though.

Reply:
ok, I added as in issue to be addressed later for now: https://github.com/jasp-stats/INTERNAL-jasp/issues/1560